src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -1079,8 +1082,16 @@ final case class EServicesApiServiceImpl(
           .leftMap(ex => InvalidZipStructure(directoryName, "Error reading configuration.json: " + ex.toString))
         importedEservice <- Either
           .catchNonFatal(jsonContent.parseJson.convertTo[ImportedEservice])
-          .leftMap(ex => InvalidZipStructure(directoryName, "Error decoding configuration.json: " + ex.toString)
-        )
+          .leftMap(ex => InvalidZipStructure(directoryName, "Error decoding configuration.json: " + ex.toString))
+        _                <- {
+          importedEservice.mode match {
+            case EServiceMode.DELIVER if importedEservice.riskAnalysis.nonEmpty =>
+              Left(InvalidZipStructure(directoryName, "riskAnalysis is not allowed in DELIVER mode"))
+            case EServiceMode.RECEIVE if importedEservice.riskAnalysis.isEmpty  =>
```

Comment:
these checks are already done on the catalog, do we need to replicate them here?